src/Microsoft.Sbom.Common/ComplianceStandard/ComplianceStandardEnforcer.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Common.ComplianceStandard;
+
+public class ComplianceStandardEnforcer
```

Comment:
2 thoughts:
1. This doesn't implement IComplianceStandardEnforcer, so we should tweak the name. Maybe `ComplianceExtensions' ?
2. This can be internal static, which would then allow `GetCommonEntityType` to be an extension method